ipn/ipnstate/ipnstate.go

```diff
@@ -27,6 +27,7 @@ import (
 // Status represents the entire state of the IPN network.
 type Status struct {
 	BackendState string
+	AuthURL      string
```

Comment:
these fields could all use some docs, but this one in particular is a bit subtle in its meaning: does non-empty only mean that we're not logged in? For which BackendState is this populated?

Reply:
Added a doc line.

It's not linked to states at all. AuthURL is populated when control sends one. I believe we only send it right now when the client is in NeedsLogin, but we could explore other possible cases. E.g. maybe we want to send a pre-emptive reauth URL sometime in the future?